front/src/common/Selector/Selector.tsx

```diff
@@ -0,0 +1,162 @@
+import cx from 'classnames';
+import { isNull } from 'lodash';
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import React, { useState, type PropsWithChildren } from 'react';
+import { GoTrash } from 'react-icons/go';
+
+import SelectNewItemButton from './SelectNewItemButton';
+import './Selector.scss';
```

Comment:
I'm not sure this is the way we have to do it, is this a reason you don't get it from /styles/scss folder?

Reply:
Done, we will rearrange all the scss files later if it's ok for you :)